app/templates/main/errorAccordion.html

```diff
@@ -0,0 +1,21 @@
+{% macro errorAccordion(tab_errors) %}.
+    <div class="govuk-accordion" data-module="govuk-accordion">
+        {%  for tab in tab_errors %}
+            <div class="govuk-accordion__section">
+                <div class="govuk-accordion__section-header">
+                    <h2 class="govuk-accordion__section-heading">
+                        <span class="govuk-accordion__section-button">{{ tab }}</span>
+                    </h2>
+                </div>
+                {%  for section in tab_errors[tab] %}
+                    <div class="govuk-accordion__section-content">
+                        <h3 class="govuk-heading-s">{{ section }}</h3>
```

Comment:
It's a little bit confusing as to what is a tab and what is a section header when looking at the page because they both look to be the same size (despite one being h2 and one h3). Perhaps make the section text smaller still?